application/src/main/webapp/vue-app/components/ExoChatApp.vue

```diff
@@ -168,10 +168,12 @@ export default {
       if (selectedContact && selectedContact.fullName && (selectedContact.room || selectedContact.user)) {
         eXo.chat.selectedContact = selectedContact;
         const indexOfRoom = this.contactList.findIndex(contact => contact.room === selectedContact.room || contact.user === selectedContact.user);
+        const index = this.contactList.indexOf(selectedContact);
```

Comment:
what is the difference between indexOfRoom and index ?

Reply:
the indexOfRoom is the index of the room containing the contacts as we can have many rooms, and the index is the rank of the contact in the list in a specific room .